enrichments/trace/internal/elastic/span.go

```diff
@@ -434,6 +462,43 @@ func (s *spanEventEnrichmentContext) enrich(
 	if cfg.ProcessorEvent.Enabled && s.exception {
 		se.Attributes().PutStr(AttributeProcessorEvent, "error")
 	}
+	if s.exceptionType == "" && s.exceptionMessage == "" {
+		// Span event does not represent an exception
+		se.Attributes().PutStr(AttributeEventKind, "event")
+		se.Attributes().PutStr(AttributeMessage, se.Name())
+		return
+	}
+
+	// Span event represents exception
+	if cfg.ErrorID.Enabled {
+		if id, err := newUniqueID(); err == nil {
+			se.Attributes().PutStr(AttributeErrorID, id)
+		}
+	}
+	if cfg.ErrorExceptionType.Enabled && s.exceptionType != "" {
+		se.Attributes().PutStr(AttributeErrorExceptionType, s.exceptionType)
+	}
+	if cfg.ErrorExceptionMessage.Enabled {
+		se.Attributes().PutStr(AttributeErrorExceptionMessage, s.exceptionMessage)
+	}
+	if cfg.ErrorExceptionHandled.Enabled {
+		se.Attributes().PutBool(AttributeErrorExceptionHandled, !s.exceptionEscaped)
+	}
+	if cfg.ErrorStacktrace.Enabled {
+		// TODO (lahsivjar): Where to parse stacktraces?
+		se.Attributes().PutStr(AttributeErrorStacktrace, s.exceptionStacktrace)
+	}
+	if cfg.ErrorGroupingKey.Enabled {
+		// See https://github.com/elastic/apm-data/issues/299
+		hash := md5.New()
+		// ignoring errors in hashing
+		if s.exceptionType != "" {
+			io.WriteString(hash, s.exceptionType)
+		} else if s.exceptionMessage != "" {
+			io.WriteString(hash, s.exceptionMessage)
+		}
+		se.Attributes().PutStr(AttributeErrorGroupingKey, hex.EncodeToString(hash.Sum(nil)))
```

Comment:
[**For reviewers**] This will be updated to include the parsed stacktrace based on above resolution.